libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1903,6 +1903,12 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 			functionType &&
 			functionType->isPure();
 
+		if (auto memberAccess = dynamic_cast<MemberAccess const*>(&_functionCall.expression()))
+			funcCallAnno.isLValue =
```

Comment:
I think this should rather use the function kind, to guard against overwrites by libraries.

Reply:
Changed to do that